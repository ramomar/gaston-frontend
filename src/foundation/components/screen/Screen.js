import React from 'react';
import { Box } from 'grommet';

function Screen({ children }) {
  return (
    <Box>
      {children}
    </Box>
  );
}

export default Screen;
