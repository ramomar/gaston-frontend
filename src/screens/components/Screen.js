import React from 'react';
import { Box } from 'grommet';

function Screen({ children }) {
  return (
    <Box
      fill='vertical'
      pad={{ horizontal: 'small' }}>
      {children}
    </Box>
  );
}

export default Screen;
