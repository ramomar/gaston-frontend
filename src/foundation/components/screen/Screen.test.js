import React from 'react';
import renderer from 'react-test-renderer';
import Screen from './Screen';

it('renders correctly', () => {

  const component =
    <Screen>
      {<div>Â¡Hola!</div>}
    </Screen>;

  const tree = renderer.create(component)

  expect(tree).toMatchSnapshot();
});
