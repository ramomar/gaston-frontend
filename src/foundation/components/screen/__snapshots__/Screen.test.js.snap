// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 tkzRw"
>
  <div>
    Â¡Hola!
  </div>
</div>
`;
